src/libs/actions/Welcome.ts

```diff
@@ -26,21 +37,21 @@ let currentUserAccountID;
  * - Whether we have loaded all reports the server knows about
  */
 function checkOnReady() {
-    if (!_.isBoolean(isFirstTimeNewExpensifyUser) || isLoadingReportData) {
+    if (isFirstTimeNewExpensifyUser === undefined || isLoadingReportData) {
         return;
     }
 
-    resolveIsReadyPromise();
+    resolveIsReadyPromise?.();
```

Comment:
I used to do `resolveIsReadyPromise && resolveIsReadyPromise()` but this seems more concise! I like the code